charts/staging/victoria-metrics/values.yaml

```
@@ -0,0 +1,39 @@
+victoria-metrics-cluster:
+  vmselect:
+    replicaCount: 5
+
+    ingress:
+      enabled: true
+      ingressClassName: nginx
+      hosts:
+        - name: select.vm.example.com
+          path: /
+          port: http
+      tls:
+        - secretName: vm-ingress-tls
+          hosts:
+            - select.vm.example.com
```

following on from @DavidFair heres how you'd do it
```suggestion
    ingress:
      enabled: true
      ingressClassName: nginx
      annotations:
          cert-manager.io/cluster-issuer: "self-signed"
      hosts:
        - name: select.vm.example.com
          path: /
          port: http
```